README.md

```diff
@@ -54,3 +54,13 @@ python examples/deformation.py
 ```
 
 That's it ;)
+
+To execute specific problem for lepl1110 command line
+it will ouput U.txt and V.txt in data folder
+
+```console
+python lepl1110.py path_to_mesh path_to_problem
+```
+
+
+
```

Comment:
````suggestion
To execute a specific LEPL1110 problem:

```console
python3 lepl1110.py path/to/mesh path/to/problem
```

This will output `U.txt` and `V.txt` in the `data` directory.
````